public/stylesheets/auth-splash.css

```diff
@@ -81,11 +81,15 @@
 }
 .auth-splash__form {
   grid-column: 1 / 3;
-  padding: 2rem 2rem 2.5rem;
+  padding: 2rem 2rem 0rem;
 }
 .auth-splash__form-group {
   position: relative;
 }
+#auth-splash__form-demo {
+  padding-top: 0px;
```

Comment:
You're using `rem`s in other places so please be sure to use `rem` here instead of pixels.